### PR TITLE
Feat:Added Toast Component

### PR DESCRIPTION
**Pull request by kiranpratyush**
updated index.html according to the documentation site(though some changes are pending for header ignore it for now updated the style.css
Below is the live working link please review it and give help full feedback

 Components Defined - 
- Toast

Live Link -  : [Testing](https://testing2311.netlify.app/Toast/)

### CODE REVIEW

File: Toast/style.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.toast {
+  padding: 1rem;
+  font-size: 1.2rem;
+  width: 400px;
+  background-color: white;
+  text-align: center;
+  border-radius: 5px;
+  border: 1px solid #4caf50;
+}
```
Comment (adidoshi) [inline Toast/style.css]: Suggestion, if you could create toast relative to the viewport, & maybe also show a demo. It would be great!

**Review by adidoshi — APPROVED** [review]
Looks fine.

File: Toast/style.css (hunk 1 above)
Comment (kiranpratyush) [reply Toast/style.css]: Thank you so much for the suggestion. I'll do

**Comment by kiranpratyush** [comment]
> Looks fine.

Thank you so much

**Review by msdeshmukh009 — COMMENTED** [review]
I think you should work on the suggestion given by @adidoshi . Maybe use an iframe to showcase the toast which is fixed to viewport or you can write JS as well.

**Comment by kiranpratyush** [comment]
> I think you should work on the suggestion given by @adidoshi . Maybe use an iframe to showcase the toast which is fixed to viewport or you can write JS as well.

I have not planned to add functionality to the toast as it is not mandatory.

**Pull request merged**